pkg/measurements/service_latency.go

```diff
@@ -186,7 +160,10 @@ func (s *serviceLatency) start(measurementWg *sync.WaitGroup) error {
 	// Reset latency slices, required in multi-job benchmarks
 	s.latencyQuantiles, s.normLatencies = nil, nil
 	defer measurementWg.Done()
-	if err := deployAssets(); err != nil {
+	svcLatencyCheckerImage := "quay.io/cloud-bulldozer/fedora-nc:latest"
+	svcLatencyCheckerCommand := []string{"sleep", "inf"}
+	err := DeployPod(types.SvcLatencyNs, types.SvcLatencyCheckerName, svcLatencyCheckerImage, svcLatencyCheckerCommand)
```

Comment:
We don't need variables if they're not used more than once.

```suggestion
	err := DeployPod(types.SvcLatencyNs, types.SvcLatencyCheckerName, "quay.io/cloud-bulldozer/fedora-nc:latest", []string{"sleep", "inf"})
```